ROADMAP.md

```diff
@@ -0,0 +1,35 @@
+# Power Platform Terraform Provider Roadmap
+
+This roadmap outlines the planned direction for the Power Platform Terraform Provider over the next 12 months. It serves as a high-level guide for contributors and users, providing transparency into where the project is headed while maintaining flexibility to adapt to community needs and evolving requirements.
+
+## Key Themes and Planned Direction
+
+| Key Theme | Planned Direction (12-month horizon) |
+|-----------|--------------------------------------|
+| **Quality & Reliability** | Increase test coverage across all resources and data sources, reduce flaky acceptance tests through improved test isolation and mock strategies, and formalize a predictable release cadence with automated quality gates. Focus on stability improvements for existing resources before adding new functionality. |
+| **Security Hardening** | Strengthen binary-signing and SBOM (Software Bill of Materials) practices in our release pipeline, integrate stricter static and dynamic analysis tools into CI/CD workflows, and address any newly reported CVEs promptly. Enhance authentication mechanisms and improve security documentation for users. |
```

Comment:
the intent of this one is really Security Improvements.   adding support for service principal auth for resources as they become available in the apis.  security hardening like SBOM could be part of this item.  Also updating release process to newer terraform registry release process.